FitGoal/FitGoal/HomeViewController.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  ViewController.swift
+//  FitGoal
+//
+//  Created by Reynaldo Aguilar on 2/3/20.
+//  Copyright © 2020 Eliany Barrera. All rights reserved.
+//
+
+import UIKit
+
+class HomeViewController: UIViewController {
+    
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        configureBackgroundView()
+    }
+}
+
+extension HomeViewController {
+    private func configureBackgroundView() {
+         view.backgroundColor = #colorLiteral(red: 0.9647058824, green: 0.9647058824, blue: 0.9647058824, alpha: 1)
+         let backgroundView = UIView(frame: CGRect(x: 0, y: 0, width: 550, height: 812))
+         backgroundView.layer.masksToBounds = true
+         backgroundView.layer.cornerRadius = 150
+         backgroundView.layer.maskedCorners = [.layerMinXMaxYCorner]
+         backgroundView.transform = backgroundView.transform.rotated(by: -0.4537856055185257)
+         backgroundView.transform = backgroundView.transform.translatedBy(x: 10, y: -600)
+
+         view.addSubview(backgroundView)
+         createGradient(in: backgroundView)
+     }
+
+     private func createGradient(in view: UIView) {
+         let gradient = CAGradientLayer()
+         gradient.frame = view.bounds
```

Comment:
You don't need to set the gradient view frame here because the frame will get outdated after the size of the view changes. You need to set its frame inside the `viewDidLayoutSubviews` function.